released/packages/coq-hierarchy-builder-shim/coq-hierarchy-builder-shim.1.2.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Cyril Cohen" "Kazuhiko Sakaguchi" "Enrico Tassi" ]
+license: "MIT"
+homepage: "https://github.com/math-comp/hierarchy-builder"
+bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
+dev-repo: "git+https://github.com/math-comp/hierarchy-builder"
+
+build: [ make "-C" "shim" "build" ]
+install: [ make "-C" "shim" "install" ]
+conflicts: [ "coq-hierarchy-builder" ]
+depends: [ "coq" ]
```

Comment:
```suggestion
depends: [ "coq" {>= "8.10"}  ]
```
This is the bound for the last release of this package. @gares or you have actually made changes to be compatible with 8.9 and below?

Reply:
No, I'll fix this upstream. Thanks